### ISSUE TITLE
Just a question

### ISSUE THREAD

**Issue by iz8mbw** [MichaelZaidman/hid-ft260#15]
Hello.
I'm interested to use an USB to I2C chip/board in order to have a "pure" I2C Master bus.
I need it in order to replace a Raspberry Pi with a Linux machine running on x86 via Proxmox.

Currently I poll I2C sensors via the I2C Master bus generated by the Raspberry Pi, all ok.

So I'm looking for an USB to I2C chip/board that can handle a "pure" I2C Master bus so I can continue to read I2C sensors.

I see that the FTDI FT260 and the Silicon Labs CP2112 are the "right" chips that can do what I need.

Both chips are supported in the Linux Kernel but it seems the Silicon Labs CP2112 have some "limitation" since when the command "i2cdetect" is executed the reply contain the message: `Warning: Can't use SMBus Quick Write, will skip some addresses`.

![image](https://github.com/MichaelZaidman/hid-ft260/assets/3474082/c9ff008d-c4ea-4a79-b088-2fd433e54c06)

So it seems the Silicon Labs CP2112 has some limitation for the addresses.

Well, does the FT260 has the same limitation?
In other words, can I use FT260 like the I2C bus generated by the Raspberry Pi?

Thanks!

**Comment by iz8mbw** [comment]
Hi Michael and thanks.
This is what "i2cdetect" can see on my bus on a Raspberry Pi (using the I2C bus generated by the Raspberry Pi SoC):

```
root@rpi3:~# i2cdetect -y 1
     0  1  2  3  4  5  6  7  8  9  a  b  c  d  e  f
00:                         -- -- -- -- -- -- -- --
10: -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --
20: -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --
30: -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --
40: -- -- -- -- 44 -- -- -- -- -- -- -- -- -- -- --
50: -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --
60: -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --
70: -- -- -- -- -- -- -- 77
root@rpi3:~#
```

If, I use the "-r" option on the same bus on a Raspberry Pi, the device "0x44" disappears:
```
root@rpi3:~# i2cdetect -r -y 1
     0  1  2  3  4  5  6  7  8  9  a  b  c  d  e  f
00:                         -- -- -- -- -- -- -- --
10: -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --
20: -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --
30: -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --
40: -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --
50: -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --
60: -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --
70: -- -- -- -- -- -- -- 77
```


So, if I use the ft260, can I see the deice (sensor) at 0x40 address and interrogate it like I do now with the Raspberry Pi?

**Comment by iz8mbw** [comment]
OK, so the "limitation" should be only at detect level and not also at reading level.
Thanks!
